packages/babel-types/src/modifications/removeProperties.ts

```diff
@@ -1,11 +1,21 @@
 import { COMMENT_KEYS } from "../constants";
 import type * as t from "..";
 
-const CLEAR_KEYS = ["tokens", "start", "end", "loc", "raw", "rawValue"];
+const CLEAR_KEYS = [
+  "tokens", // only exist in t.File
+  "start",
+  "end",
+  "loc",
+  // Fixme: should be extra.raw / extra.rawValue?
+  "raw",
+  "rawValue",
```

Comment:
It seems to be a bug, or they are for compatibility issues that I am not aware of.

Reply:
This looks like a bug.